src/libraries/Common/src/Interop/Windows/SspiCli/ISSPIInterface.cs

```diff
@@ -15,6 +15,7 @@ internal interface ISSPIInterface
         int EnumerateSecurityPackages(out int pkgnum, out SafeFreeContextBuffer pkgArray);
         int AcquireCredentialsHandle(string moduleName, Interop.SspiCli.CredentialUse usage, ref SafeSspiAuthDataHandle authdata, out SafeFreeCredentials outCredential);
         int AcquireCredentialsHandle(string moduleName, Interop.SspiCli.CredentialUse usage, ref Interop.SspiCli.SCHANNEL_CRED authdata, out SafeFreeCredentials outCredential);
+        unsafe int AcquireCredentialsHandle(string moduleName, Interop.SspiCli.CredentialUse usage, Interop.SspiCli.SCH_CREDENTIALS* authdata, out SafeFreeCredentials outCredential);
```

Comment:
Why use a pointer instead of a ref (what the rest of the methods do)?

Reply:
I could net get ref through lambda used to inpersonificate.